pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -1015,42 +1020,41 @@ struct ArgSortOpConversion : public OpConversionPattern<tile::ArgSortOp> {
       minVal = rewriter.create<LoadOp>(loc, minValVar, zeroIndex).getResult();
       // Is compVal smaller than minVal? If so, update the allocs
       Value orderPred;
-      if (elementType.isSignedInteger()) {
-        CmpIPredicate dir{};
+      if (elementType.isa<FloatType>()) {
+        CmpFPredicate dir{};
```

Comment:
Why is there a trailing `{}`?

Reply:
Actually I didn't add this but just reordered the cases. Anyway it looks unnecessary so I will remove it.